src/Form/FormHelperText.js

```diff
@@ -0,0 +1,70 @@
+// @flow weak
+/* eslint-disable jsx-a11y/label-has-for */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from '../utils/customPropTypes';
+
+export const styleSheet = createStyleSheet('MuiFormHelperText', (theme) => {
+  return {
+    root: {
+      color: theme.palette.input.helperText,
+      fontSize: 12,
+      lineHeight: 1.33333333,
```

Comment:
@oliviertassinari In Chrome 1.3333333 lineheight = 16px height, 1.333333 = 14 px 😄  I set it 1.4 also gives 16px but don't know how in can affect browsers when using retina display.